playpen/repo_level_awareness/agent/dependency_agent/api.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+# trunk-ignore(bandit/B405)
+import xml.etree.ElementTree as ET
+from dataclasses import dataclass
+
+from langchain_core.messages import HumanMessage
+
+
+@dataclass
+class FQDNResponse:
+    artifact_id: str
+    group_id: str
+    version: str
+
+    def to_llm_message(self) -> HumanMessage:
+        return HumanMessage(f"the result is {json.dumps(self.__dict__)}")
```

Comment:
Does the `__dict__` have any cruft in it or just what you want?

Reply:
IIUC `self.__dict__` just makes the data class a dict, so yes everything in the dataclass is what I want

If I am wrong please let me know, this was a lot of google -> code as I try and re-learn python

Reply:
looks like that's accurate if you only have strings/no nested values. There's also `dataclass.asdict` which handles edge cases but this is probably fine